src/schema/paddle.ts

```diff
@@ -131,7 +131,7 @@ export const typeDefs = /* GraphQL */ `
     pricePreviews: PricePreviews! @auth
     corePricePreviews: PricePreviews! @auth
     pricingMetadata(type: PricingType): [ProductPricingMetadata!]! @auth
-    pricingPreview(type: PricingType): [ProductPricingPreview!]! @auth
```

Comment:
@capJavert also removed the `@auth` directive since we don't require auth for requesting the prices anymore. For example, on Funnel.